README.md

```diff
@@ -1,6 +1,6 @@
-# To-Do-list-list-structure
+# To Do list: add & remove
 
-In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.
+In this project, you will implement the CRUD (create, read, update, delete) methods. All the elements of the user interface will be fully functional and your application will be completed.
 
 ## Built With
 
```

Comment:
- [ ] OPTIONAL You can use this structure to style your links in the README file: [link name](link URL), as an example [Live Demo](and here paste your link) aka https://kal2944.github.io/To-dolist/ - try it and you will see how your repo will look more pretty 👍 

- You can use it everywhere, where you have links 💯